### PR TITLE
Removed popup package and created my own popup system

### PR DESCRIPTION
**Pull request by jagger-harris**
Removed the temporary system for pop ups and implemented a more standard approach. Also added the ability to close a pop up and restructured code.

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -46,7 +37,6 @@ function Navbar() {
   return (
     <>
       <div className="flex justify-between p-5">
-        {accounts.length !== 0 && network !== POLYGON_CHAIN_ID ? <SwitchMessagePopup/> : null}
```
Comment (victor-wei126) [inline src/components/Navbar.jsx:49]: why was this removed and how was the logic reencoded elsewhere?

**Review by victor-wei126 — COMMENTED** [review]
I took a brief skim and I doubt it'll cause problems. But the code I wrote for this is badly coupled with other stuff so I might take another closer look. In the meantime can you provide more explanation of the code you moved around?

File: src/components/Navbar.jsx (hunk 1 above)
Comment (jagger-harris) [reply src/components/Navbar.jsx:49]: The entire `SwitchMessagePopup` component was deleted as it was unnecessary during the refactor. That particular logic is now in the function `selectConnectPopup` on line 158.

**Comment by jagger-harris** [comment]
> I took a brief skim and I doubt it'll cause problems. But the code I wrote for this is badly coupled with other stuff so I might take another closer look. In the meantime can you provide more explanation of the code you moved around?

File: src/components/Card.jsx [hunk 2]
```diff
@@ -7,27 +7,26 @@ import VoteRatio from "./VoteRatio";
 import { setSelectedProposal } from "../../reduxActions";
 import { useDispatch } from 'react-redux/es/exports';
 
-function Card(props) {
-
+function Card({proposal}) {
```
Comment (austin-davis1) [inline src/components/Card.jsx:10]: Destructuring props is good here since we only have one prop passed in

File: src/components/CardSection.jsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ function CardSection(props) {
     }
 
     return (
-      data.map(proposal => <Card proposal={proposal}/>)
+      data.map(proposal => <Card key={proposal.metadata.title} proposal={proposal}/>)
```
Comment (austin-davis1) [inline src/components/CardSection.jsx:21]: Good call here with adding a key, the warnings are annoying.  On a future note, we may want to add keys anywhere else in the code that we may have mappings (not sure if we have them anywhere else tho)

File: src/pages/Landing.jsx [hunk 4]
```diff
@@ -4,16 +4,16 @@ import { Link } from "react-router-dom";
...
-          <div className="flex flex-col justify-center items-center">
-            <h1 className="display-1 animate-character">University DAO</h1>
-            <Link reloadDocument className="mr-5 w-72 h-10 flex justify-center items-center rounded-lg text-2xl bg-purple hover:bg-hover-purple" to={"/dashboard"}>Launch App</Link>
-          </div>
+  return (
+    <div className="landing-page">
+      <div className="landing-content">
+        <div className="flex flex-col justify-center items-center">
+          <h1 className="display-1 animate-character">University DAO</h1>
+          <Link onClick={reload} className="transition-all duration-200 mr-5 w-72 h-10 flex justify-center items-center rounded-lg text-2xl bg-purple hover:bg-hover-purple" to={"/dashboard"}>Launch App</Link>
```
Comment (austin-davis1) [inline src/pages/Landing.jsx:12]: What is this reload function?

Comment (jagger-harris) [reply src/pages/Landing.jsx:12]: Oh good call. That was used to force reload the page but led to an infinite loop so I deleted it. I will fix that.
